jujugui/static/gui/src/app/jujulib/test-bakery.js

```diff
@@ -105,6 +105,11 @@ describe('Bakery', () => {
       });
     });
 
+    it('properly handles cookie auth', () => {
+      bakery.sendRequest('http://example.com/set-auth-cookie', 'PUT');
+      assert.deepEqual(client._sendRequest.args[0][6], true);
```

Comment:
Maybe one more send/assertion in this test to check that it properly sets it to false otherwise.

Also this doesn't really show what it is testing, could you add a comment "checking for withCredentials" or something along those lines.